pulser/_seq_drawer.py

```diff
@@ -52,12 +57,12 @@ def gather_data(seq):
                 if slot.type == 'target':
                     target[(slot.ti, slot.tf-1)] = slot.targets
                 continue
-            pulse = slot.type
+            pulse = cast(Pulse, slot.type)
             if (isinstance(pulse.amplitude, ConstantWaveform) and
                     isinstance(pulse.detuning, ConstantWaveform)):
                 time += [slot.ti, slot.tf-1]
-                amp += [pulse.amplitude._value] * 2
-                detuning += [pulse.detuning._value] * 2
+                amp += [int(pulse.amplitude._value)] * 2
+                detuning += [int(pulse.detuning._value)] * 2
```

Comment:
```
pulser\_seq_drawer.py:65: error: List item 0 has incompatible type "float"; expected "int"
pulser\_seq_drawer.py:66: error: List item 0 has incompatible type "float"; expected "int"
```

Reply:
This makes sense, it's because the first things being added to `amp` and `det` are `[0]`, so it is being implicitly typed as `list[int]`. If you change all the `0` to `0.`, it should fix this.